runtime/commonTest/src/kotlinx/serialization/test/TestHelpers.kt

```diff
@@ -9,7 +9,7 @@ import kotlinx.serialization.internal.EnumSerializer
 import kotlin.test.*
 
 @Suppress("TestFunctionName")
-inline fun <reified E : Enum<E>> EnumSerializer(serialName: String): EnumSerializer<E> =
+internal inline fun <reified E : Enum<E>> EnumSerializer(serialName: String): EnumSerializer<E> =
```

Comment:
it is function in test sources anyway

Reply:
it exposes internal `EnumSerializer` and should be internal